tests/pubsub/test_mcache.py

```diff
@@ -4,6 +4,8 @@
 
 
 class Msg:
+    __slots__ = ["topicIDs", "seqno", "from_id"]
```

Comment:
May I ask the reason why we need `__slots__` here?

Reply:
to make flake8 happy "B903".